src/balances/taxExporter.ts

```diff
@@ -0,0 +1,67 @@
+import { BigNumber } from 'bignumber.js';
+import { curry } from 'lodash';
+import { combineLatest, Observable } from 'rxjs';
+import { flatMap, switchMap } from 'rxjs/operators';
+import { proxyAddress$ } from '../blockchain/calls/instant';
+import { NetworkConfig } from '../blockchain/config';
+import { vulcan0x } from '../blockchain/vulcan0x';
+import { web3 } from '../blockchain/web3';
+
+export interface TaxExportItem {
```

Comment:
What TaxExportItem and TaxExport is for?

Reply:
You are right. It's not necessary - not in use anymore. I will remove it.